django_mongodb/client.py

```diff
@@ -4,11 +4,45 @@
 
 
 class DatabaseClient(BaseDatabaseClient):
-    executable_name = "mongo"
+    executable_name = "mongosh"
 
     @classmethod
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
-        raise NotImplementedError
+        options = settings_dict.get("OPTIONS", {})
```

Comment:
Generally, the OPTIONS dict will correspond to the optional parameters of [MongoClient](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html#pymongo.mongo_client.MongoClient).

We don't need to support every mongosh option in this wrapper, but if there's a corresponding MongoClient parameter, then it could be okay.

Reply:
Perfect, I want to support:

1. Uri
2. authentication database and mechanism
3. retry_writes

all the others aren't implemented (like AWS token)

Reply:
Okay, so it looks like these should be specified as OPTIONS as camelcase, right? (e.g. retryWrites)

Reply:
I thought we had to follow the snake case :grimacing: (No matter if the other part uses comelcase). So yeap, they should be in camelcase.

Reply:
We want the user to fill out the OPTIONS dictionary based on the kwargs that MongoClient accepts. All this backend does is the equivalent of: 
`MongoClient(..., **settings_dict['OPTIONS'])`
(see `DatabaseWrapper._connect()`)